### DISCUSSION TITLE
Why is logging in so difficult?

### DISCUSSION THREAD

**Discussion by thojanssens** [mastodon/mastodon#17439] [General]
Hello,

So I created an account. Now when I head to https://joinmastodon.org/ there is no login link/button...

Seriously I created an account and I have no idea to login into it.... terrible UX. When I click "Get started" I need to choose a community, which I did already when registering; so I suppose I have to browse through the community to find the one I registered at? If I even remember because it was just a general community. And there isn't any search function by community name?
I don't get it.

Note that I don't need help, I can sort it all out; just sharing the experience of a newly registered and frustrated user.

Sorry for the aggressive writing<3

**Comment by Gargron** [comment]
joinmastodon.org is for joining. Once you register on a server (e.g. example.com), everything is between you and example.com. Login there. If you don't remember where you signed up, look for the welcome e-mail you got after signing up, it will link to the right place.

**Comment by thojanssens** [comment]
@Gargron thank you Gargron, I kinda guessed that. But I just personally think that we can do a little better. I'm not fan of this process/flow. Thank you for clarifying.

**Comment by extratone** [comment]
I think we could reasonably call this hurdle one of the classics for newcomers to the Fediverse - it confused me too, at first, but I'd like to stop by to make what might actually just be an idiosyncratic argument.

Setting aside *the feasibility dilemma* of another arrangement for onboarding across the Fediverse, generally, I would suggest the experience as it is now - requiring that new users determine that they must make a (virtually consequencesless) choice between a selection of instances *separate* a given fedi project's landing page - is actually an ideal means of educating them about the fundamentals of how these networks work. (And therefore, yes, how they differ from traditional Big Social services.)

In other words, I don't think the project should seek to remove any more of the steps required for onboarding from the process as it stands, currently. With respect, the mentality that lends one to recoil at the idea of traversing a few different web domains in order to join a social "s̷e̷r̷v̷i̷c̷e̷" (protocol) is a *direct* result of our normalization of the very worst bits of the centralized, adtech-driven web. Just wait until you see a PixelFed post looking completely native in the federated Mastodon timeline for the first time hehe.

Whatever rigamarole remains is natural law for user experiences with real, meaningful choice involved.
